src/build/genprod.scala

```diff
@@ -172,19 +175,13 @@ object Function {
 class Function(val i: Int) extends Group("Function") with Arity {
   def descriptiveComment  = ""
   def functionNTemplate =
-"""
+s"""
```

Comment:
the introduction of the `s` here seems unnecessary? it makes me go looking for uses of `$` and `${...}`, but none are present